crates/sui-indexer-alt-framework/src/pipeline/mod.rs

```diff
@@ -3,6 +3,7 @@
 
 use std::time::Duration;
 
+pub use crate::handlers::cp_mapping::PrunableRange;
```

Comment:
This is kind of a random place to re-export this from -- for context the other re-exports from this module exist because the modules being re-exported from are private to the crate.